cogs/games/league_of_legends/lolclashmodule.py

```diff
@@ -83,7 +83,7 @@ async def clash_add(self, ctx, availability: Optional[str], *roles):
             beebot_profiles_data = beebot_profiles.get_beebot_profiles_json()
             # add member's roles
             beebot_profiles_data = beebot_profiles.beebot_profile_exists(
-                available_member, beebot_profiles_data)
+                beebot_profiles_data, available_member)
```

Comment:
I think the arguments are the wrong way around, so I just included the bugfix in this PR. Let me know if I'm reading it wrong though!

Reply:
Your fix is correct! The arguments in cogs.helper.helper_functions usually go in the order of how they are accessed in the dictionary. For example; arguments go (data, key) because the dictionary is accessed like data[key].